src/08-Security.mdx

```diff
@@ -0,0 +1,95 @@
+---
+name: 08. Security
+---
+
+import Authors from './helpers/Authors';
+
+# Security
+Level: *Hidden Hippopotamus*
+
+The security aspects of your project are just as important as performance and UX.
+Although more security issues refer to the Backend it's important for any Frontend Developer to know at least the theory behind all those security issues.
+
+A very good place to start is the free and open software security community: The [OWASP Foundation](https://www.owasp.org/).
+Every couple of years they publish a list of the most critical frontend and backend security risks, the **Top Ten List**.
+> A powerful awareness document for web application security that represents a broad consensus about the most critical security risks to web applications
+
+Download latest Top Ten PDF here: https://www.owasp.org/images/7/72/OWASP_Top_10-2017_%28en%29.pdf.pdf
+
+
+## Golden Rule for Frontend Development
+
+**NEVER EVER LEAVE INCOMING DATA UNATTENDED!!!**
```

Comment:
it is not the incoming data per se it is always the combination of incoming user data AND outputting / using it anywhere in your frontend code. for example a form field which is just sent to backend and not used / outputtet in frontend is not an issue for FE.

Reply:
is this addition really important? isn't it self-evident that data you never use is not critical?

do you have a short and concise addition for that sentence?

Reply:
not sure, it's onboarding. Maybe for a junior it is not?